### PR TITLE
apps: disable reporting min/avg/max group uptime by default

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

The min/avg/max group uptime charts are not really useful (unlike "carried over time uptime"), I think we need to disable them by default.

This PR disables reporting those charts by default. They can be enabled using `with-detailed-uptime` command-line option.

##### Component Name

`collectors/apps.plugin`

##### Test Plan

Run the plugin w/ and w/o `with-detailed-uptime`.

### CODE REVIEW

**Comment by thiagoftsm** [comment]
@ilyam8 code is working as expected, but before I approve this PR, I will suggest we change apps.plugin documentation saying what happens when we enable this option.

**Comment by ilyam8** [comment]
@thiagoftsm I've checked the apps.plugin documentation and I see we don't really describe all the available options. We just briefly mention that there are some, use `--help` for more details.

See the end of the [Configuration](https://github.com/netdata/netdata/tree/master/collectors/apps.plugin#configuration) section. Moreover, as I mentioned in the OP, I consider min/avg/max uptime functionality very questionable and not really useful.

**Comment by thiagoftsm** [comment]
> @thiagoftsm I've checked the apps.plugin documentation (did it before marking the PR ready for review) and I see we don't really describe all the available options. We just briefly mention that there are some, use `--help` for more details.
> 
> See the end of the [Configuration](https://github.com/netdata/netdata/tree/master/collectors/apps.plugin#configuration) section. Moreover, as I mentioned in the OP, I consider min/avg/max uptime functionality very questionable and not really useful.

Yes you are right, `apps.plugin` is going in a different direction that we have for `ebpf.plugin`. @DShreve2 probably this is something we will need to change too.

**Review by thiagoftsm — DISMISSED** [review]
Everything worked as expected. LGTM!

File: collectors/apps.plugin/apps_plugin.c [hunk 1]
```diff
@@ -3519,26 +3520,28 @@ static void send_collected_data_to_netdata(struct target *root, const char *type
...
+      }
+      send_END();
+
+      send_BEGIN(type, "uptime_max", dt);
+      for (w = root; w ; w = w->next) {
+          if(unlikely(w->exposed && w->processes))
+              send_SET(w->name, w->uptime_max);
+      }
+      send_END();
     }
```
Comment (vlvkobal) [inline collectors/apps.plugin/apps_plugin.c:3544]: Please, fix the indentation.

**Review by vlvkobal — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**